### DISCUSSION TITLE
手机singbox没有速度无法连接，其他客户端没试过因为我用v2ray

### DISCUSSION THREAD

**Discussion by 14790897** [vvbbnn00/WARP-Clash-API#170] [Q&A]
![Screenshot_2024-04-14-21-05-57-658_io.nekohasekai.sfa.jpg](https://github.com/vvbbnn00/WARP-Clash-API/assets/121866954/29fe02df-a72f-4da1-adeb-25cb9c1424ac)

**Comment by 14790897** [comment]
http://15.165.158.82:21001/

密码123456

大佬求助 @vvbbnn00

**Reply by vvbbnn00** [reply]
链接似乎无法访问

**Reply by 14790897** [reply]
大佬，现在可以访问了，之前安装了甬哥的warp脚本，好像会导致ip端口无法访问，就是下面这个
```sh
bash <(wget -qO- https://gitlab.com/rwkgyg/CFwarp/raw/main/CFwarp.sh 2> /dev/null)

```

**Comment by 14790897** [comment]
今天试了一下clash  verge也不能使用
<img width="1880" height="1400" alt="image" src="https://github.com/user-attachments/assets/ba303d8f-83bf-4c89-9fea-e1ace6304524" />
